firefox-manifest.json

```diff
@@ -30,5 +30,10 @@
 			],
 			"js": ["firefox/main.js"]
 		}
-	]
-}
\ No newline at end of file
+	],
+	"browser_specific_settings": {
+		"gecko": {
+			"id": "{89ae2b95-4a1f-4ad5-b401-1d9911dc530d}"
```

Comment:
extension id needs to be updated to `"{119be3f3-597c-4f6a-9caf-627ee431d374}"` so it does not conflict with existing extension when downloaded

Reply:
thinking about this more, it might be easier to just merge this as-is and create another commit on top to replace the id. it looks like you want to publish your own copy of the extension and this would make it easier to maintain both repositories, if desired

Reply:
Nah don't worry I'm not trying to create a fork, I just wanted to let people who are less deep in coding/GitHub know how they can use these changes right away without waiting for the addon stores to publish them - the extension ID was just a way to mute a warning in Firefox that happens if there is none!